### PR TITLE
WIP: One Operator per Database

### PR DESCRIPTION
**Pull request by eberlep**

### CODE REVIEW

**Comment by eberlep** [comment]
Still missing: Cleanup of old namespaces (after deletion of operator)

File: controllers/postgres_controller.go [hunk 1]
```diff
@@ -12,6 +12,7 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/ViaQ/logerr/log"
```
Comment (majst01) [inline controllers/postgres_controller.go]: This import looks weird

Comment (eberlep) [reply controllers/postgres_controller.go]: Yup, sorry. I hate the auto imports on save in VS Code :-)

**Comment by majst01** [comment]
CRD modification is missing

**Review by majst01 — APPROVED** [review]

**Pull request merged**